src/libmongoc/tests/test-mongoc-handshake.c

```diff
@@ -979,4 +1297,23 @@ test_handshake_install (TestSuite *suite)
    TestSuite_Add (suite,
                   "/MongoDB/handshake/race_condition",
                   test_mongoc_handshake_race_condition);
+   TestSuite_Add (suite, "/MongoDB/handshake/faas/valid_aws", test_valid_aws);
```

Comment:
```suggestion
   TestSuite_AddFull (suite,
                      "/MongoDB/handshake/faas/valid_aws",
                      test_valid_aws,
                      NULL /* dtor */,
                      NULL /* ctx */,
                      test_framework_skip_if_no_setenv);
```

Skip test if test runner is unable to set environment variables. This may fix the Windows tasks. Suggestion applies to all new tests in this file.